scripts/source-maintenance.sh

```diff
@@ -721,13 +723,25 @@ collectAuthors ()
         return 0 # successfully did nothing, as requested
     fi
 
-    vettedCommitPhraseRegex='[Rr]eference point for automated CONTRIBUTORS updates'
+    vettedPhraseRegex='[Rr]eference point for automated CONTRIBUTORS updates'
+
+    vettedByPr=false
+    if test -n "${PULL_REQUEST_NUMBER}"
+    then
+        vettedByPr=`gh pr view $PULL_REQUEST_NUMBER --json body --jq ".body | test(\"${vettedPhraseRegex}\")"`
```

Comment:
Please adjust this code to fail (i.e. return a non-zero error code from this function, possibly based on `$?`) if `gh` command itself fails. In other words, let's require that `gh` succeeds when PULL_REQUEST_NUMBER is set.

Reply:
Adjusted.